htdocs/config2/settings-dist-common.inc.php

```diff
@@ -318,7 +318,6 @@ function cronjobs_enabled()
     global $opt;
 
     return $opt['cron']['debug'] ||
-    // @file_get_contents($opt['page']['absolute_http_url'] . 'register.php') !== false;
```

Comment:
This code may me improved (by adding user/pw data) and re-enabled. It's not obsolete but incomplete.